rust/src/core.rs

```diff
@@ -19,11 +19,12 @@ pub struct TempEtcGuard {
     renamed_etc: bool,
 }
 
+/// Detect if we have /usr/etc and no /etc, and rename if so.
 pub(crate) fn prepare_tempetc_guard(rootfs: i32) -> CxxResult<Box<TempEtcGuard>> {
     let rootfs = ffi_view_openat_dir(rootfs);
-    let has_usretc = rootfs.exists("usr/etc")?;
+    let has_etc = rootfs.exists("etc")?;
```

Comment:
Do we care about the case where `/etc` exists as an empty directory? That flow used to work before the patch and will be silently skipped after this, but I'm not sure if anything else does create a "placeholder etc mountpoint" at some point.

Reply:
We don't do that anywhere in rpm-ostree.  I can't think of a good reason to do that outside of it either.